### PR TITLE
Add Linux Masterclass Course - Beginner to Advanced

### LINKED ISSUE #296: [FEATURE] Add Linux Masterclass Course - Beginner to Advanced
### Description

Add Linux Masterclass Course - A complete tutorial from Beginner to Advanced from Apoorv Goyal to the DevOps roadmap in the Linux section.

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by abhishekchandra2522k**
## Fixes Issue

Closes #296 

## Changes proposed

Add the Linux masterclass playlist link to the DevOps and Backend development roadmaps (README.md).

## Note to reviewers

Added the playlist link in the Linux section for both DevOps and Backend development roadmaps, some indentation and spaces were added/removed from the prettier extension in VS Code, but output remains the same!

### CODE REVIEW

File: Backend-Development/README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 - [CSS](https://www.youtube.com/watch?v=1Rs2ND1ryYc)
 - [HTML](https://www.youtube.com/watch?v=kUMe1FH4CHE)
 
 ### Build a project
+
 - [frontendpractice.com](https://www.frontendpractice.com/)
 - [frontendmentor.io](https://www.frontendmentor.io/)
 
 ## Understanding Internet
+
```
Comment (siddhant-khisty) [inline Backend-Development/README.md:22]: Please remove the blank line 22.

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]
Looks good. Just one small linting error

File: Backend-Development/README.md (hunk 1 above)
Comment (abhishekchandra2522k) [reply Backend-Development/README.md]: Done, removed the blank line 22.

**Review by siddhant-khisty — APPROVED** [review]
LGTM

**Pull request merged**